includes/admin/class-wcmp-country-codes.php

```diff
@@ -8,254 +8,24 @@
-        "DZ",
-        "VI",
-        "AO",
-        "AG",
-        "AR",
-        "AM",
-        "AW",
-        "AU",
-        "AZ",
-        "BS",
-        "BH",
-        "BD",
-        "BB",
-        "BZ",
-        "BJ",
-        "BM",
-        "BT",
-        "BO",
-        "BW",
-        "BR",
-        "VG",
-        "BN",
-        "BF",
-        "BI",
-        "KH",
-        "CA",
-        "KY",
-        "CF",
-        "CL",
-        "CN",
-        "CO",
-        "KM",
-        "CG",
-        "CD",
-        "CR",
-        "CU",
-        "DJ",
-        "DM",
-        "DO",
-        "EC",
-        "EG",
-        "SV",
-        "GQ",
-        "ER",
-        "ET",
-        "FK",
-        "FJ",
-        "PH",
-        "GF",
-        "PF",
-        "GA",
-        "GB",
-        "GM",
-        "GE",
-        "GH",
-        "GD",
-        "GP",
-        "GT",
-        "GN",
-        "GW",
-        "GY",
-        "HT",
-        "HN",
-        "HK",
-        "HR",
-        "IN",
-        "ID",
-        "IS",
-        "IQ",
-        "IR",
-        "IL",
-        "CI",
-        "JM",
-        "JP",
-        "YE",
-        "JO",
-        "CV",
-        "CM",
-        "KZ",
-        "KE",
-        "KG",
-        "KI",
-        "KW",
-        "LA",
-        "LS",
-        "LB",
-        "LR",
-        "LY",
-        "MO",
-        "MG",
-        "MW",
-        "MV",
-        "MY",
-        "ML",
-        "MA",
-        "MQ",
-        "MR",
-        "MU",
-        "MX",
-        "MN",
-        "MS",
-        "MZ",
-        "MM",
-        "NA",
-        "NR",
-        "NP",
-        "NI",
-        "NC",
-        "NZ",
-        "NE",
-        "NG",
-        "KP",
-        "UZ",
-        "OM",
-        "TL",
-        "PK",
-        "PA",
-        "PG",
-        "PY",
-        "PE",
-        "PN",
-        "PR",
-        "QA",
-        "RE",
-        "RU",
-        "RW",
-        "KN",
-        "LC",
-        "VC",
-        "PM",
-        "WS",
-        "ST",
-        "SA",
-        "SN",
-        "SC",
-        "SL",
-        "SG",
-        "SO",
-        "LK",
-        "SD",
-        "SR",
-        "SZ",
-        "SY",
-        "TJ",
-        "TW",
-        "TZ",
-        "TH",
-        "TG",
-        "TO",
-        "TT",
-        "TD",
-        "TN",
-        "TM",
-        "TC",
-        "TV",
-        "UG",
-        "UY",
-        "VU",
-        "VE",
-        "AE",
-        "US",
-        "VN",
-        "ZM",
-        "ZW",
-        "ZA",
-        "KR",
-        "AN",
-        "BQ",
-        "CW",
-        "SX",
-        "XK",
-        "IM",
-        "MT",
-        "CY",
-        "CH",
-        "TR",
-        "NO",
-    ];
-
     /**
-     * @param string $countryCode
-     *
      * @return bool
      */
-    public static function isEuCountry(string $countryCode): bool
+    public static function isEuCountry(): bool
     {
-        return in_array($countryCode, self::EURO_COUNTRIES);
+        return (new AbstractConsignment())->isEuCountry();
```

Comment:
Hoe komt dit automagisch te weten over welk land het gaat?

Reply:
Eerder in de code wordt de method `setCountry` gebruikt waardoor het land bekend is. In de `isEUCountry` wordt het land opgehaald door `getCountry`.

Reply:
dan geloof ik dit :P